.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,100 @@
+name: E2E Tests
+
+on:
+  deployment_status:
+
+jobs:
+  e2e-tests:
+    if: |
+      github.event_name == 'deployment_status' &&
+      github.event.deployment_status.state == 'success' &&
+      (
+        (
+          contains(github.event.deployment.environment, 'Production') &&
+          contains(github.event.deployment_status.target_url, 'liam-erd-web')
+        ) ||
+        (
+          contains(github.event.deployment.environment, 'Preview') &&
+          !contains(github.event.deployment_status.target_url, 'liam-erd-sample') &&
+          !contains(github.event.deployment_status.target_url, 'liam-docs')
+        )
+      )
```

Comment:
The preview environment is difficult to identify because of URLs like ``https://vercel.com/route-06-core/liam-app/BUkhUNMkptDdh5dL1D3DQui9A8Vw`` 😞.